src/material.ts

```diff
@@ -55,33 +62,43 @@ export const defaultMaterialConfig = {
     },
 };
 
-export class Material extends HasConsistencyChecksHasMetadataNamedDefaultableInMemoryEntity {
-    constructor(config) {
-        super(config);
-        this._json = lodash.cloneDeep(config || {});
-    }
+interface Property {
```

Comment:
Should be from ESSE too

Reply:
DerivedPropertiesSchema doesn't have `name` property
Should I add it to to esse schema?

Reply:
Resolved